### ISSUE TITLE
Not able to Identify the links under "In this article" as links: A11y_VisualStudioCodeServices_Uninstall Visual Studio Code_In this article_Info and Relationships.

### ISSUE THREAD

**Issue by prabhudevu** [microsoft/vscode-docs#7415]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**

## GitHub Tags:
#A11yTCS; #A11ySev3; #A11yMAS; #DesktopWeb; #ChromiumEdge; #BM_VisualStudioCodeServices_Web_July2024; #Visual Studio Code Services; #Win11; #FTP; #Desktopweb; #WCAG1.3.1; #Element:Link

## Environment Details: 
Application Name:  Visual Studio Code Services
URL: https://code.visualstudio.com/docs/setup/uninstall?dark-plus-v2#_windows
Microsoft Edge version 126.0.2592.68 (Official build) (64-bit)

## Repro steps:
1. Launch the application URL: https://code.visualstudio.com/docs/setup/uninstall?dark-plus-v2#_windows and press ENTER.
2. "Uninstall Visual Studio Code" screen gets opened.
3. TAB to Links under "In this article"
4. Observe that not able to Identify the links under "In this article" as links.

## Actual:
Not able to Identify the links under "In this article" as links.

Note: Issue is repro with below URLs as well
https://code.visualstudio.com/docs/setup/uninstall?dark-plus-v2
https://code.visualstudio.com/docs/supporting/faq?dark-plus-v2
https://code.visualstudio.com/docs?dark-plus-v2
https://code.visualstudio.com/download?dark-plus-v2
https://code.visualstudio.com/insiders/?dark-plus-v2
https://code.visualstudio.com/?dark-plus-v2
https://code.visualstudio.com/docs/python/editing?dark-plus-v2
https://code.visualstudio.com/docs/python/jupyter-support-py?dark-plus-v2
https://code.visualstudio.com/docs/python/linting?dark-plus-v2
https://code.visualstudio.com/docs/remote/remote-overview?dark-plus-v2
https://code.visualstudio.com/docs/remote/troubleshooting?dark-plus-v2
https://code.visualstudio.com/docs/remote/wsl?dark-plus-v2
https://code.visualstudio.com/docs/?dv=win64user&dark-plus-v2
https://code.visualstudio.com/docs/editor/command-line?dark-plus-v2
https://code.visualstudio.com/docs/getstarted/introvideos?dark-plus-v2
https://code.visualstudio.com/docs/getstarted/keybindings?dark-plus-v2
https://code.visualstudio.com/docs/getstarted/locales?dark-plus-v2
https://code.visualstudio.com/docs/java/java-debugging?dark-plus-v2
https://code.visualstudio.com/docs/languages/csharp?dark-plus-v2
https://code.visualstudio.com/docs/python/debugging?dark-plus-v2

## Expected
Links under "In this article" should be defined in a way for us to understand they are links.

## User Impact
Users will not be able to know that the items under in this article" as links.

## Attachment
![1linkey](https://github.com/microsoft/vscode-docs/assets/98821792/f39c8c22-c277-4331-97cf-4341a6ed9c88)

![1  Note artile](https://github.com/microsoft/vscode-docs/assets/98821792/cb77bae6-129c-45dd-a40d-0fcf385c378d)

![Screenshot 2024-06-26 183900](https://github.com/microsoft/vscode-docs/assets/98821792/c8a6c5da-a93e-4f01-ad93-6ebce1f05367)

**Comment by daviddossett** [comment]
The links in the table of contents and "In this article" section are technically links, but they aren't links that need to be differentiated from paragraph text. I don't think it makes sense to decorate these in the same way we've done for the inline links with the grey underline. It would be a total mess of underlines. 

I suggest we don't make any changes here yet given that this isn't new behavior for the v2 website design.

**Comment by 12-shweta** [comment]
#rev:shwarh

**Comment by 12-shweta** [comment]
#A11yPreview;

**Comment by rzhao271** [comment]
Hi @prabhudevu, considering that the left sidebar doesn't have underlined links, is there a way we could format the right sidebar instead to not require underlined links?

**Comment by 12-shweta** [comment]
@rzhao271 in the below way also we can differentiate check the right pane
![image](https://github.com/user-attachments/assets/59b8c3bd-ad18-43d9-801f-5793f98a2d88)

**Comment by rzhao271** [comment]
Hi @12-shweta, I have added underlines _on hover_ to match the behaviour of the right navbar links with that of the left navbar links. I am still not interested in introducing always-visible underlines to the right navbar links unless there are other users confused about those links.

**Comment by rzhao271** [comment]
Closing as wontfix. See https://github.com/microsoft/vscode-docs/issues/7415#issuecomment-2427757216. and https://github.com/microsoft/vscode-docs/issues/7415#issuecomment-2660547365.
